examples/README.md

```diff
@@ -17,7 +17,7 @@ Have a look at Yew's [starter templates](https://yew.rs/docs/getting-started/sta
 git clone https://github.com/yewstack/yew.git
 cd yew/examples
 ./build.sh minimal # example subfolder
-python3 -m http.server --directory static # open localhost:8000 in browser
+cd static && python3 -m http.server  # open localhost:8000 in browser
```

Comment:
I'm not entirely sure what the benefit of this change is.

Reply:
This one is because the `--directory` flag is Python 3.7+ and was failing on my older Python 3.6.